raiden/tasks.py

```diff
@@ -110,16 +105,10 @@ def _run(self):  # pylint: disable=method-hidden
                         time.time() - self.protocol.last_received_time[receiver_address]
                     )
                     # Add a randomized delay in the loop to not clog the network
-                    gevent.sleep(randint(0, int(0.2 * self.send_ping_time)))
+                    gevent.sleep(random.randint(0, int(0.2 * self.send_ping_time)))
```

Comment:
The random module was being [imported twice](https://github.com/raiden-network/raiden/blob/ef5bf6a8cb81646092393ddb09b05d70da8e6db1/raiden/tasks.py#L4-L6)

From the two options I could remove either `import random` or `from random import randint`, I happened to prefer the explicit version.